mobile/src/main/java/org/openhab/habdroid/background/BackgroundTasksManager.kt

```diff
@@ -46,8 +46,8 @@ class BackgroundTasksManager : BroadcastReceiver() {
                 NotificationUpdateObserver.createNotificationChannels(context)
             }
             ACTION_RETRY_UPLOAD -> {
-                val retryInfos = intent.getParcelableArrayListExtra<RetryInfo>(EXTRA_RETRY_INFOS)
-                for (info in retryInfos) {
+                val retryInfo = intent.getParcelableArrayListExtra<RetryInfo>(EXTRA_RETRY_INFO)
+                for (info in retryInfo) {
```

Comment:
Old spelling seems more correct (it's a list, hence plural makes more sense)

Reply:
Yes, plural is better here, but the plural of information is information.

Reply:
OK, so how about either `retryInfoList` or simply
```
for (info in intent.getParcelableArrayListExtra<RetryInfo>(EXTRA_RETRY_INFO_LIST))
```
(same line of thought applies below)?

I really don't want variable names to be misleading, even if the misleading name is linguistically more correct.